packages/cadl-ranch-specs/http/special-headers/request-id/main.tsp

```diff
@@ -0,0 +1,27 @@
+import "@typespec/http";
```

Comment:
I think it would be better to have the folder structure as `headers/request-id`. Thoughts @timotheeguerin ?

Reply:
`request-id` shall be a special header of azure service. I prefer either put it under `azure/request-id` or keep the current `special-headers`.

Reply:
hard to say if it is that standard compared to repatibility headers or not. We can keep it there for now but maybe rename to the full header name `client-request-id